doc/classes/ProjectSettings.xml

```diff
@@ -194,6 +194,9 @@
 		<member name="application/boot_splash/use_filter" type="bool" setter="" getter="" default="true">
 			If [code]true[/code], applies linear filtering when scaling the image (recommended for high resolution artwork). If [code]false[/code], uses nearest-neighbor interpolation (recommended for pixel art).
 		</member>
+		<member name="application/boot_splash/display_time_msec" type="int" setter="" getter="" default="0">
```

Comment:
I think the setting should be minimum_display_time, not display time. It really depends on how long does Godot take to initialize on a given platform.